src/screens/Technology/components/Details/index.tsx

```diff
@@ -279,7 +282,39 @@ export const Documents = () => {
         presentationStyle="overFullScreen"
       />
 
-      {false && <S.CostSection>Vídeos</S.CostSection>}
+      {technology.attachments.documents.length ? (
```

Comment:
Here we will have unexpected behavior. The length will be displayed on the screen. How to fix 👇 
```suggestion
      {!!technology.attachments.documents.length ? (
```